config/local/bigquery.yml.tmpl

```diff
@@ -1,5 +1,5 @@
 method: oauth
 keyfile: ""
-dataset: "{{ var('username') }}_private_working_schema"
+dataset: {{"{{ var('username') }}_private_working_schema"}}
```

Comment:
This was needed due to errors from jinja 
https://stackoverflow.com/questions/25060537/jinja2-escape-sequence-of-interpreted-characters